### PR TITLE
Remove and search notes

### PR DESCRIPTION
**Pull request by AyushP90**
#### Short description of what this resolves:
Delete notes and search for them using a substring
#### Changes proposed in this pull request:

- Can remove notes now
- Searching is done using substring matching
- Temporary fix to 42

**Fixes**: #42

### CODE REVIEW

File: modules/diary.py [hunk 1]
```diff
@@ -237,7 +237,63 @@ def complete_task():
         chalk.red(
```
Comment (manparvesh) [inline modules/diary.py]: please remove the above .swp files

File: resources/vocab-words.txt [hunk 2]
```diff
@@ -676,3 +676,5 @@ parlous - full of danger or uncertainty; precarious
 pilloried - attack or ridicule publicly; pillory is a wooden shackle
 expatiate - speak or write at length or in detail
 man - someone who serves in the armed forces; a member of a military force
+work - make uniform
```
Comment (manparvesh) [inline resources/vocab-words.txt]: please remove the changes from this file

File: modules/diary.py [hunk 3]
```diff
@@ -247,14 +303,16 @@ def notes():
...
+            click.echo("S.No. | Time   | Note")
+            click.echo("------|--------|-----")
 
+            counter=0;
             for entry in contents['entries']:
+                counter+=1
                 time = entry['time']
                 text = entry['text']
-                click.echo(time + "| " + text)
+                click.echo("--" + str(counter) + "---|" + time + "| " + text)
```
Comment (manparvesh) [inline modules/diary.py]: please reformat the files you've changed using `autopep8` after you're finished working on them

File: modules/diary.py [hunk 4]
```diff
@@ -237,7 +237,63 @@ def complete_task():
...
+                    click.echo(time + "| " + text)
+                    found=1
+
+            if found==0:
+                click.echo('No notes found having the above substring')
+
+    else:
+        chalk.red(
+            'There are no notes for today. Add a new note by entering "yoda diary nn"')
+
```
Comment (manparvesh) [inline modules/diary.py]: Sorry, this is not what I meant by searching the notes. I would like to give the user an option to search in the previous notes. There is no point in searching today's notes, right?

**Comment by AyushP90** [comment]
Thanks I'll work on it.

**Pull request closed without merging**